src/App.tsx

```diff
@@ -3,12 +3,14 @@ import "typeface-roboto";
 import { Router } from "@reach/router";
 import SignIn from "./SignIn";
 import Calendar from "./Calendar";
+import Project from "./Project";
 
 function App() {
   return (
     <Router>
       <SignIn path="/" />
       <Calendar path="/calendar" />
+      <Project  path="/project" />
```

Comment:
I'm picturing this as a dialog within the calendar, but open to discussion about it.